src/Entity/Events.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Drupal\civicrm_entity\Entity;
+
+use Drupal\Core\Annotation\PluralTranslation;
+use Drupal\Core\Annotation\Translation;
+use Drupal\Core\Entity\Annotation\ContentEntityType;
+
+/**
+ * Defines the comment entity class.
+ *
+ * @ContentEntityType(
```

Comment:
Do we need this? Shouldn't the `hook_entity_type_build()` replace the need for an annotation?